feature_engine/selection/feature_selection.py

```diff
@@ -0,0 +1,176 @@
+import numpy as np
+from sklearn.base import BaseEstimator, TransformerMixin
+from sklearn.ensemble import RandomForestClassifier
+from sklearn.metrics import get_scorer
+from sklearn.model_selection import cross_validate
+from sklearn.utils.validation import check_is_fitted
+
+from feature_engine.dataframe_checks import (
+    _is_dataframe,
+    _check_input_matches_training_df,
+)
+from feature_engine.variable_manipulation import (
+    _define_variables,
+    _find_numerical_variables,
+)
+
+
+class FeatureSelection(BaseEstimator, TransformerMixin):
+    """
```

Comment:
could we please rename the class to UnivariatePerformanceSelection() or SignleFeaturePerformanceSelection() not sure which one is best. Maybe the second?